apps/estuary/test/integration/application_start_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Estuary.StartTest do
+  use ExUnit.Case
+  use Divo
+
+  @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
+  @event_stream_topic Application.get_env(:estuary, :event_stream_topic)
+
+  setup do
+    case Application.ensure_all_started(:estuary) do
+      {:ok, _apps} -> :ok
```

Comment:
do we need this block anymore now that we're using divo?

Reply:
I would say this is a nice to have for now if we don't want to run our integration tests using divo.

Reply:
My fear is it getting copied and pasted in other places.  I wont strong arm you into deleting it, but if it ends up in more than 1 file, I will probably put in a PR to delete it.

Reply:
not sure i understand the comment; you're going to run your tests with Divo by virtue of including `use Divo` in the header of the file.
unless you explicitly say `--no-start` when you run the mix task to prevent the VM from starting your app, they will always be started. this setup is not buying you anything in the integration environment.

Reply:
It seems like by including `use divo` it starts `docker-compose` and does the health check. This was adding extra time when running the tests locally if kafka doesn't start up right away which seems to be very sporadic. What I was doing locally was commenting out the `use divo` manually starting the containers and just letting them run so I could run my integration tests multiple times without having to start and stop the containers and do the health check. To do that though you have to make sure the full supervisor tree starts which is what this is doing. I believe divo is doing something similar as well.